forte/processors/misc/remote_processor.py

```diff
@@ -67,14 +68,40 @@ def initialize(self, resources: Resources, configs: Config):
                 f"{response.status_code}: "
                 "Remote service not started or invalid endpoint configs."
             )
+        service_name: str = response.json()["name"]
+
+        if self.configs.do_validation:
```

Comment:
After the discussion with Jenny, it seems it is a better idea to implement this validation at the processor level. We can keep this validate service name part here since this is special.

The config name can also be changed to something more meaningful, instead of `do_validation`, it can be `do_init_type_check`